Update the pages on transport configs (#531)
wso2/docs-mi: en/docs/reference/synapse-properties/transport-parameters/mailto-transport-parameters.md

> # MailTo Parameters

Reviewer: ```suggestion
# MailTo parameters
```

> When you implement an integration use case that requires the capability of sending emails (over SMTP) or receiving emails (over POP3 or IMAP), you can use the following parameters in your [proxy service]({{base_path}}/develop/creating-artifacts/creating-a-proxy-service) artifact.

Reviewer: ```suggestion
When implementing an integration use case that requires the capability to send emails (via SMTP) or receive emails (via POP3 or IMAP), you can use the following parameters in your [proxy service]({{base_path}}/develop/creating-artifacts/creating-a-proxy-service) artifact.
```

!!! Info

>     The Micro Integrator can use the MailTo transport only if the transport listener and sender are enabled and configured at the server level. Read about the [MailTo transport]({{base_path}}/install-and-setup/setup/transport-configurations/configuring-transports/#configuring-the-mailto-transport).

Reviewer: ```suggestion
    MailTo transport can be used only if the transport listener and sender are enabled and configured at the server level. Read about the [MailTo transport]({{base_path}}/install-and-setup/setup/transport-configurations/configuring-transports/#configuring-the-mailto-transport).
```
